hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeSchemaOneDBDefinition.java

```diff
@@ -91,7 +91,7 @@ protected DatanodeSchemaOneDBDefinition(String dbPath) {
 
   @Override
   public DBColumnFamilyDefinition[] getColumnFamilies() {
-    return new DBColumnFamilyDefinition[] {getBlockDataColumnFamily(),
-        getMetadataColumnFamily(), getDeletedBlocksColumnFamily() };
+    return new DBColumnFamilyDefinition[] {getMetadataColumnFamily(),
```

Comment:
Does this mean we cannot use this tool to access metadata and deletedblocks data from Schema V1 DB?

Reply:
With respect to the current patch yes , it can only access  blockData.  It's also possible to access metadata if needed. I faced an error while trying to access  the DeletedBlocks i.e `Invalid chunk information. This data may have been written using datanode schema version one, which did not save chunk information.`